README.md

```diff
@@ -2,103 +2,103 @@
 
-```
+This model, specifically the Gazebo plugin that implements the control
+interface, requires Gazebo 9, which is the default version for ROS
+Melodic.
 
-### Setup your workspace:
-#### A. Create a catkin workspace:
-```
-source /opt/ros/kinetic/setup.bash
-mkdir -p ~/catkin_ws/src
-cd ~/catkin_ws/
-catkin build
-source devel/setup.bash
-```
-For more information, visit [create_a_workspace](http://wiki.ros.org/catkin/Tutorials/create_a_workspace)
+## Building
 
-#### B. Initialize your catkin workspace:
-Navigate to the root of your catkin workspace, if you are not already with `cd ~/catkin_ws`.
-Initialize your workspace:
+Make sure all required dependecies are installed using
+[`rosdep`](http://wiki.ros.org/rosdep). From the top directory of your
+catkin workspace run (assuming this repository is cloned directly
+under `src`):
 ```
-catkin init
+rosdep update && rosdep install --from-paths src/sd_twizymodel/ --ignore-src -r -y
 ```
 
-#### C. Clone this repository or copy its contents at your `~/catkin_ws/src` folder of the catkin workspace you just initialized.
-#### D. Navigate to your workspace and build the simulation
-```
-cd ~/catkin_ws
-catkin build sd_robot sd_control sd_description
-```
-If you have previously built your workspace with catkin_make:
-Either clean your workspace with `catkin clean` and rebuild with `catkin build`
-or build the SD Twizy Gazebo packages in isolation with `catkin_make --pkg sd_robot sd_control sd_description`.
-After the built has successfully finished, do:
-```
-source devel/setup.bash
-```
-#### E. Launch the simulation:
-This launches the vehicle model in Gazebo and RViz for visualizing the sensors' output.
+Now you can build the packages in this repository. `catkin_make`,
+[`catkin_tools`](https://catkin-tools.readthedocs.io/en/latest/installing.html)
+and
+[`colcon`](https://catkin-tools.readthedocs.io/en/latest/installing.html)
+should all work.
+
+### Autoware
+
+The `sd_autoware` package included in this repository provides an
+interface to be able to control the StreetDrone model using the
+[Autoware.AI](https://catkin-tools.readthedocs.io/en/latest/installing.html)
+open-source self-driving car stack.
+
+By default this package is not built to prevent any problems when
+Autoware.AI is not available. If it is available and you do want to
+use the package, remove the file
+`streetdrone_model/sd_autoware/CATKIN_IGNORE` and build again.
+
+## Launch the simulation
+The following launches the vehicle model in Gazebo and RViz for
+visualizing the sensors' output:
+
 ```
-roslaunch sd_robot sd_twizy_empty.launch
+roslaunch sd_bringup sd_twizy_gazebo.launch
 ```
 
-<p align="center"> 
-<img src="streetdrone_model/sd_docs/imgs/sd.png">
+<p align="center"> <img src="streetdrone_model/sd_docs/imgs/sd.png">
 </p>
 
-You might need to update your ignition-math version `sudo apt upgrade libignition-math2`
+There are several more launch files included to provide a more
```

Comment:
The main issue I think is https://bitbucket.org/osrf/gazebo/issues/2607/error-restcc-205-during-startup-gazebo, that models never load because the model server has issues. I will add instructions to fix that by using https://api.ignitionfuel.org/ as model server.

The model.config file errors can be ignored, they are shown for the other launch files because they run gazebo in verbose mode (and gazebo looks for models in places it shouldn't). If that's turned on for the empty launch file as well, it too shows those errors. I will turn off verbose in the launch files.

Reply:
Tested with the latest changes and all seem to work, after changing the model server URL